validatedeps.js

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env node
```

Comment:
Yes, I can combine them and enhance checkdeps a little. You think the command args should be a list of projects to validate?

./validatedeps.js blueocean-dashboard blueocean-personalization weird-folder/oddly-nested-project-for-some-odd-reason ?